src/compiler/compiler.cc

```diff
@@ -331,35 +332,39 @@ void Compiler::language_server(const Compiler::Configuration& compiler_config) {
   Filesystem* fs = null;
   LspFsProtocol* fs_protocol = null;
   LspFsConnection* connection = null;
+  LspWriter* writer;
```

Comment:
The initialization code below is complex enough that I'd want to initialize this to null here.

Reply:
done.